auxiliary/auxiliary.go

```diff
@@ -232,13 +244,13 @@ func SaveSecrets(secrets map[string]interface{}) {
 	}
 
 	// Write the JSON to the disk.
-	ioutil.WriteFile("./.pocket/secrets", []byte(jsonFormattedSecrets), 0700)
+	ioutil.WriteFile("/home/awn/.pocket/secrets", []byte(jsonFormattedSecrets), 0700)
```

Comment:
Change this.

Import `os/user` to get `u := user.Current()` and then use `u.HomeDir`. See https://godoc.org/os/user#User.